pkg/internalapis/cnsvolumeoperationrequest/cnsvolumeoperationrequest.go

```diff
@@ -120,22 +107,124 @@ func InitVolumeOperationRequestInterface(ctx context.Context) (VolumeOperationRe
 // Returns an error if any error is encountered while attempting to
 // read the previously persisted information from the API server.
 // If no previous information has been persisted, it returns an empty object.
```

Comment:
if instance not found then it returns nil.

Reply:
Removed this comment since we return error now for all errors